tests/lib/k8s/test_builders.py

```diff
@@ -42,6 +42,24 @@ def check_namespace(parameters):
             "namespace": check_namespace,
         }
 
+    @patch("nestor_api.lib.k8s.builders.io", autospec=True)
+    def test_load_templates(self, io_mock):
+        """Should load the templates and correctly substitute `{{variable}}`"""
```

Comment:
Following PEP, docstring should end with a period (I know, it is quite strict...)